README.md

```diff
@@ -28,6 +28,7 @@ The path where `brew` will be installed.
     homebrew_installed_packages:
       - ssh-copy-id
       - pv
+      -  { name: vim, install_options: "with-luajit,override-system-vi" }
```

Comment:
Could you also add a sentence after the current description that mentions that `install_options` is optional, and if used, you need to explicitly set the `name` for the package as well? Something to that effect.